core-bundle/templates/Frontend/preview_toolbar_base.html.twig

```diff
@@ -18,7 +18,7 @@
             <div>
                 <button type="button" id="copyPublishedPath" class="tl_submit">{{ 'MSC.copyURL'|trans([], 'contao_default') }}</button>
                 {% if share is not empty %}
-                    <a href="{{ share }}" target="_blank" class="tl_submit" onclick="window.open(this.href+'&amp;url='+location.href);return false">{{ 'MSC.shareURL'|trans([], 'contao_default') }}</a>
+                    <a href="{{ share }}" target="_blank" class="tl_submit cto-toolbar__share-url">{{ 'MSC.shareURL'|trans([], 'contao_default') }}</a>
```

Comment:
JavaScript inline event handlers are not allowed when using a `script-src` or `script-src-attr` policy. Thus this was moved to the `preview_toolbar_base_js.html.twig` template.